### PR TITLE
tuxedo-keyboard: rename to tuxedo-drivers and update to 4.5.0

### LINKED ISSUE #47715: Package request: tuxedo-drivers is the new tuxedo-keyboard
### Package name

tuxedo-drivers

### Package homepage

https://github.com/tuxedocomputers/tuxedo-drivers

### Description

The repository for tuxedo-keyboard is archived and the new one is called tuxedo-drivers.

With this change they've also changed the creation process for a package.

I couldn't get my head around and hope somebody else with more knowledge knows how to do it.

### Does the requested package meet the package requirements?

System, Compiled

### Is the requested package released?

Yes

### PR DESCRIPTION
**Pull request by orchy-contrib**
closes #47715

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture (x86_64-musl)

### CODE REVIEW

File: srcpkgs/tuxedo-drivers/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+do_install() {
+	sed "s/#MODULE_VERSION#/${version}/" debian/tuxedo-drivers.dkms > src/dkms.conf
+	vmkdir usr/src/${pkgname}-${version}
+	vcopy src/* usr/src/${pkgname}-${version}
+	vinstall tuxedo_keyboard.conf 644 usr/lib/modprobe.d/
+}
+
+tuxedo-keyboard_package() {
+	depends="tuxedo-drivers"
```
Comment (Luciogi) [inline srcpkgs/tuxedo-drivers/template:23]:
```suggestion
	depends="tuxedo-drivers>=${version}_${revision}"
```

Comment (orchy-contrib) [reply srcpkgs/tuxedo-drivers/template]: fixed

**Comment by orchy-contrib** [comment]
Updated to 4.5.0 (still works)

**Comment by newbluemoon** [comment]
Sorry I didn’t catch this. Please give me a ping next time. :)

**Pull request merged**